src/yang/drned-xmnr.yang

```diff
@@ -312,9 +312,19 @@ module drned-xmnr {
               type filepath-pattern-type;
               mandatory true;
             }
-            leaf overwrite {
-              type boolean;
-              default false;
+            choice conflicts {
```

Comment:
This deserves a grouping, I think - it is a verbatim copy of the choice for `import-states`, isn't it?